packages/flutter_tools/test/integration/test_driver.dart

```diff
@@ -52,30 +54,27 @@ class FlutterTestDriver {
     return msg;
   }
 
-  // TODO(dantup): Is there a better way than spawning a proc? This breaks debugging..
```

Comment:
Is this comment no longer relevant?

Reply:
When I first wrote it, it bugged me that I couldn't debug into it correctly but I discussed with Devon and we decided that spawning in-process would be bad because a) it wouldn't exercise all of the code (and the intention of these tests is to cover as much as possible) and b) it would potentially leave state hanging around (like the Contexts) between the tests which could give misleading results or result in flakes.